ui/components/SidebarComponent.tsx

```diff
@@ -60,7 +60,7 @@ export default function SidebarComponent(props: { logout?: boolean }) {
               />
             </div>
           </div>
-          {props.logout && <Logout />}
+          {<Logout />}
```

Comment:
```suggestion
          <Logout />
```
The goal here was to only render logout button if auth is enabled